### PR TITLE
Misc copy edits

### PR DESCRIPTION
**Pull request by chasballew**
cc @sandersonet @mixonic @fancyremarker

### CODE REVIEW

File: app/components/pricing-page-header/template.hbs [hunk 1]
```diff
@@ -1,6 +1,7 @@
-{{#if isDevelopment}}
+{{!-- {{#if isDevelopment}}
   <h1>Add a Payment Method to Your Pay-as-you-go Development Plan</h1>
 {{/if}}
 {{#if isPlatform}}
   <h1>Add a Payment Method to Your PHI-ready Platform Plan</h1>
-{{/if}}
```
Comment (mixonic) [inline app/components/pricing-page-header/template.hbs]: Can this code just be removed instead of commented?

File: app/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Diesel</title>
```
Comment (mixonic) [inline app/index.html]: :+1:

**Comment by chasballew** [comment]
@mixonic - Removed that pricing header.

**Comment by chasballew** [comment]
@mixonic - Rebased and passing

**Comment by mixonic** [comment]
Fantastic, thanks :-)

**Pull request merged**